### PR TITLE
use specific component in SignTransaction flow

### PR DESCRIPTION
**Pull request by chabroA**
### 📝 Description

The `SignTransaction` flow is more or less a duplicate of the `Send` flow.
Porting the use of coin specific component to this flow as well.
Added note on tech debt to clean up. Tracked here on Jira as well -> https://ledgerhq.atlassian.net/browse/LIVE-9033

Related to this PR https://github.com/LedgerHQ/ledger-live/pull/3708

### ❓ Context

- **Impacted projects**: LLD
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-8684

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

Screenshot 2023-08-22 at 12 11 08

### 🚀 Expectations to reach

When signing a tx through a live app, the appropriate network fee wording should be displayed, as well as the tooltip besides it (cf. screenshot above).

This should be the case for tx signed from live-apps using the legacy `live-app-sdk` (like lido for example) as well as for tx signed from live-apps using the `wallet-api` (like wallet-connect)

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/modals/SignTransaction/NOTES.md
Comment (haammar-ledger) [inline apps/ledger-live-desktop/src/renderer/modals/SignTransaction/NOTES.md:1]: :100:

**Review by haammar-ledger — APPROVED** [review]
Wondering if there are playwright tests covering the sign transaction flow, but probably not...

**Comment by chabroA** [comment]
> Wondering if there are playwright tests covering the sign transaction flow, but probably not...

Well, it looks like there are some, since some tests failed -> https://github.com/LedgerHQ/ledger-live/actions/runs/5937471400/job/16099962376 🕵️

**Pull request merged**